File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.js
function solution(a, b) {
    
    if(a == b) return a;
    
    let min = Math.min(a, b);
    let max = Math.max(a, b);
    
    var answer = 0;
    
    for(let i=min; i<=max; i++){
        answer += i;
    }
    
    return answer;
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        
        if(a == b) return a;
        
        int from = Math.min(a, b);
        int to = Math.max(a, b);
        
        long answer = 0;
        
        for(int i=from; i<=to; i++){
            answer += i;
        }
        
        return answer;
    }
}